src/Valit/ValitatorExtensions.cs

```diff
@@ -5,16 +5,10 @@ namespace Valit
 {
     public static class ValitRulesProviderExtensions
```

Comment:
Could you change the name of this class to ``ValitRulesExtensions`` since now there's no provider ? :D

Reply:
You're right! commit incoming...

Reply:
Hmm the file itself is named `ValitatorExtensions`. Do you have a preference? 
I would go for `ValitRulesExtensions`.

Reply:
I totally agree. The name should be ``ValitRulesExtensions``